lldb/include/lldb/Breakpoint/BreakpointLocation.h

```diff
@@ -369,6 +393,13 @@ class BreakpointLocation
   lldb::break_id_t m_loc_id; ///< Breakpoint location ID.
   StoppointHitCounter m_hit_counter; ///< Number of times this breakpoint
                                      /// location has been hit.
+  std::optional<LineEntry> m_preferred_line_entry; // If this exists, use it to print the stop
```

Comment:
IMO the end-of line comment style only makes sense if the comment fits on one line.
```suggestion
  /// If this exists, use it to print the stop description rather than the LineEntry
  /// ...
  std::optional<LineEntry> m_preferred_line_entry; 
```

Reply:
And if you use the Doxygen comment syntax, IDEs will know to associate the comment with the following declaration.